reference/marketplace-api.yaml

```diff
@@ -2,7 +2,7 @@ openapi: 3.0.0
 servers:
   - url: 'https://developers.vendasta.com/api/v1'
     description: Production Server
-  - url: 'https://marketplace-proxy-demo.vendasta-internal.com/api/v1'
+  - url: 'https://developers-demo.vendasta.com/api/v1'
     description: Demo Server
```

Comment:
I wonder if we should keep the old domain as a 3rd option. That way vendors that integrated against the old domain can find the docs that match their code.

Reply:
do we need to add third option as **Old Demo Server:https://marketplace-proxy-demo.vendasta-internal.com/api/v1**?

Reply:
[In Slack](https://vendasta.slack.com/archives/CPZLDHE9K/p1685393561381449?thread_ts=1685132220.281849&cid=CPZLDHE9K) @ncline-va said not to bother with the Legacy server. We will keep just the new one.